docs/recipes/endpoint-testing-with-mongoose.md

```diff
@@ -156,3 +139,13 @@ And you're done!
 You may choose to abstract code for `test.before`, `test.beforeEach`, `test.afterEach.always` and `test.after.always` into a separate file.
 
 To see a demo of this configuration file, look at https://github.com/zellwk/ava-mdb-test
+
+## Why `test.serial` instead of `test`
```

Comment:
That does make sense but since ava is meant to be run in parallel it'd be better to explain how to set it up by default instead of expecting tests to run in serial.